src/effgen.ml

```diff
@@ -200,6 +200,32 @@ module StaticGenerators = struct
                   (recgen (n / 2)) )
             ])
   ;;
+
+  let all_type_gen =
```

Comment:
Where is `all_type_gen` used in the rest of the program? Or are you only using it for testing?

Reply:
It is for testing now but will be useful when we transition to generating terms for all types present in Efftester.